AltStore/Operations/FetchTrustedSourcesOperation.swift

```diff
@@ -11,9 +11,9 @@ import Foundation
 private extension URL
 {
     #if STAGING
-    static let trustedSources = URL(string: "https://f000.backblazeb2.com/file/altstore-staging/altstore/trustedsources.json")!
+    static let trustedSources = URL(string: "https://raw.githubusercontent.com/Spidy123222/SideStore/change-source/trustedapps.json")!
```

Comment:
This still says Spidy’s repo

Reply:
This is currently for testing. Once confirmed it works then can change it to sidestore.

Reply:
> This still says Spidy’s repo

I have been able to confirm it works I’ll make necessary changes.

Reply:
Changes are now directed to Sidestore Repo